apps/docs/pages/guides/getting-started/quickstarts/solidjs.mdx

```diff
@@ -87,21 +87,17 @@ export const meta = {
 
       ```jsx src/App.jsx
         import { createClient } from "@supabase/supabase-js";
-        import { createEffect, createSignal, For } from "solid-js";
+        import { createResource, For } from "solid-js";
 
         const supabase = createClient('https://<project>.supabase.co', '<your-anon-key>');
 
+        async function getCountries() {
+          const { data } = await supabase.from("countries").select();
+          setCountries(data);
```

Comment:
```suggestion
          return data;
```

Reply:
wow went completely over my head, thanks for mentioning 😆